alongtheway.txt

```diff
@@ -203,13 +203,13 @@ Reflect: Our reading today starts with Isaac and I don't want to skip over Isaac
 
 Respond:
 What did you notice about Jacob as you read?
-Did you notice any repeating themes?
+Did you notice any recurring themes?
```

Comment:
I prefer repeating to recurring but as I look at this more closely "repeating themes" seems redundant. It could probably just be simplified to "themes"

Sent from my iPhone

> On Jan 15, 2016, at 10:12 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> > @@ -203,13 +203,13 @@ Reflect: Our reading today starts with Isaac and I don't want to skip over Isaac
> >  
> >  Respond:
> >  What did you notice about Jacob as you read?
> > -Did you notice any repeating themes?
> > +Did you notice any recurring themes?
> like this word?
> 
> —
> Reply to this email directly or view it on GitHub.
>